gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/CgmesExportService.java

```diff
@@ -109,58 +104,38 @@ Map<String, ByteArrayOutputStream> createAllSshFiles(Network mergedNetwork, SweD
             Country country = entry.getKey();
             String tso = entry.getValue();
             if (subnetworksByCountry.containsKey(country)) {
-                mapSshFiles.putAll(createOneSsh(subnetworksByCountry.get(country), sweData, tso));
+                LOGGER.info("Building cgmes files for country {}", country);
+                mapFiles.putAll(createTsoFiles(subnetworksByCountry.get(country), sweData, tso));
             }
         }
-        return mapSshFiles;
+        return mapFiles;
     }
 
-    private Map<String, ByteArrayOutputStream> retrieveEqAndTpFiles(SweData sweData) throws IOException {
-        LOGGER.info("Retrieving EQ & TP files");
+    private Map<String, ByteArrayOutputStream> createTsoFiles(Network network, SweData sweData, String tso) throws IOException {
         Map<String, ByteArrayOutputStream> mapFiles = new HashMap<>();
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.RTE_TP));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.REE_TP));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.REN_TP));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.RTE_EQ));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.REE_EQ));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.REN_EQ));
+        exportParams.put(CgmesExport.PROFILES, CGMES_PROFILES);
```

Comment:
That's not exactly what I had in mind.
As the values we put in the exportParams never change, there is no need to perform this action each time you want to create TSO files: do it only once when the application starts and never do it afterwards.

Here is what I was talking about:
```java
@Service
public class CgmesExportService {
    private static final Logger LOGGER = LoggerFactory.getLogger(CgmesExportService.class);

    private static final List<String> CGMES_PROFILES = List.of("EQ", "TP", "SSH");
    private static final Properties TSO_FILES_EXPORT_PARAMS = new Properties();
    private static final Properties COMMON_FILES_EXPORT_PARAMS = new Properties();
    private static final DateTimeFormatter CGMES_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMdd'T'HHmm'Z'_'[process]_[tso]_[type]_001.xml'");

    private final Logger businessLogger;
    private final FileExporter fileExporter;
    private final UrlValidationService urlValidationService;

    static {
        TSO_FILES_EXPORT_PARAMS.put(CgmesExport.PROFILES, CGMES_PROFILES);
        TSO_FILES_EXPORT_PARAMS.put(CgmesExport.EXPORT_BOUNDARY_POWER_FLOWS, true);

        COMMON_FILES_EXPORT_PARAMS.put(CgmesExport.PROFILES, "SV");
        COMMON_FILES_EXPORT_PARAMS.put(CgmesExport.EXPORT_BOUNDARY_POWER_FLOWS, true);
    }

    public CgmesExportService(Logger businessLogger, FileExporter fileExporter, UrlValidationService urlValidationService) {
        this.businessLogger = businessLogger;
        this.fileExporter = fileExporter;
        this.urlValidationService = urlValidationService;
    }

[...]


    private Map<String, ByteArrayOutputStream> createTsoFiles(Network network, SweData sweData, String tso) throws IOException {
        Map<String, ByteArrayOutputStream> mapFiles = new HashMap<>();
        MemDataSource memDataSource = new MemDataSource();
        network.write("CGMES", TSO_FILES_EXPORT_PARAMS, memDataSource);
        for (String profile : CGMES_PROFILES) {
            putAndRenameFile(network.getNameOrId(), sweData, tso, memDataSource, mapFiles, profile);
        }
        return mapFiles;
    }

[...]

    Map<String, ByteArrayOutputStream> createCommonFile(Network network, SweData sweData) throws IOException {
        LOGGER.info("Building SV file");
        try (ByteArrayOutputStream os = new ByteArrayOutputStream()) {
            MemDataSource memDataSource = new MemDataSource();
            network.write("CGMES", COMMON_FILES_EXPORT_PARAMS, memDataSource);
            String filenameFromCgmesExport = network.getNameOrId() + "_SV.xml";
            os.write(memDataSource.getData(filenameFromCgmesExport));
            String outputFilename = buildCgmesFilename(sweData, "CGMSWE", "SV");
            return Map.of(outputFilename, os);
        }
    }

[...]
```